app/src/main/java/org/oppia/android/app/testing/SpotlightFragmentTestActivity.kt

```diff
@@ -20,7 +20,7 @@ class SpotlightFragmentTestActivity : TestActivity() {
 
     spotlightFragmentTestActivityPresenter.handleOnCreate(
       intent.getIntExtra(
-        PROFILE_ID_ARGUMENT_KEY, -1
+        PROFILE_ID_ARGUMENT_KEY, /* profileIdKEyDefaultValue */ -1
```

Comment:
```suggestion
        PROFILE_ID_ARGUMENT_KEY, /* profileIdKeyDefaultValue */ -1
```

Ditto below for line 38.

Reply:
done